### PR TITLE
chore(tools): change .eslintrc to be compatible with eslint 2.0

### PR DESCRIPTION
**Pull request by akath19**
ESLint 2.0 removed the rules "space-after-keywords" and "space-return-throw-case" and replaced them with "keyword-spacing", said rule controls spacing behaviour for spaces before and after keywords, .eslintrc has been changed to reflect that and now spaces are necessary before and after each keyword

BREAKING CHANGES: compatibility with ESLint < 2.0

(had to create a new PR, I accidentally broke the repo on the previous one)

### CODE REVIEW

**Comment by EisenbergEffect** [comment]
Thanks for bearing with us! Sorry for the extra work this caused you.

File: .eslintrc [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
-  "root": true,
```
Comment (gheoan) [inline .eslintrc:2]: Why is this line removed?

**Comment by akath19** [comment]
@Gheoan sorry, I took an earlier .eslintrc that didn't have your PR, I've added it back.

**Comment by EisenbergEffect** [comment]
Sorry for the delay! PR is merged and tools are released as 0.2.0

**Comment by EisenbergEffect** [comment]
@Akath19 We have merged this in...but after attempting to update one of our repos to eslint 2.x we cannot get any form of linting to work. It appears to not work with Babel or not recognize es2015 at all. Would you be willing to help us convert one of our repos to eslint 2.x so we can see how that is done? After that, we can update the rest of our repos. A good candidate repo would be `http-client`. Let me know if you can assist with this. We've tried on several occasions and are unable to get it to work or find any documentation to assist us.

**Comment by akath19** [comment]
@EisenbergEffect No problem, I'll download http-client and figure out what´s wrong and how can we fix it.

**Comment by EisenbergEffect** [comment]
I haven't pushed any attempts to fix it, so it's still using the old version. If you can help us in this way, it would be much appreciated. Thanks!

**Comment by akath19** [comment]
@EisenbergEffect  I've been at this for a while (with the http-client repo) and eslint always works whether it's local or globally installed, I've tested in both Win10 and OSX, so I don't think its an env issue.

Is this still not working for you?

**Comment by EisenbergEffect** [comment]
We've got it working now. Thanks!

**Comment by EisenbergEffect** [comment]
(There was a bug in one specific version of eslint which was resolved.)

**Comment by akath19** [comment]
Ok, I was getting worried that it was one of those weird untrackable errors.

Glad everything is working now!
